nvm.sh

```diff
@@ -2417,6 +2416,7 @@ nvm() {
         local VERSION_PATH
         VERSION_PATH="$(nvm_version_path "${VERSION}")"
         local EXIT_CODE
+        EXIT_CODE=-1
```

Comment:
similarly, this should be initialized at the first definition of the var, after line 2408.